src/app/pages/HomePage/Graph/HelpScreen/index.tsx

```diff
@@ -102,29 +110,21 @@ const HelpScreen: FC<HelpScreenProps> = ({ setParaTimeStep }) => {
     <HelpScreenContainer>
       <SwiperBox>
         <swiper-container style={{ height: '100%' }} ref={swiperElRef} slides-per-view="1">
-          {allSteps.map((step, index) => (
+          {allSteps.map(({ icon, label }, index) => (
             <swiper-slide
               style={{ height: '100%', display: 'flex', justifyContent: 'center', alignItems: 'center' }}
-              key={step.label}
+              key={label}
             >
-              {Math.abs(activeStep - index) < totalSteps ? (
-                <Box
-                  component="img"
-                  sx={theme => ({
-                    display: 'block',
-                    height: 50,
-                    overflow: 'hidden',
-                    margin: `0 auto ${theme.spacing(3)}`,
-                  })}
-                  src={step.icon}
-                  alt={step.label}
-                />
-              ) : null}
+              {Math.abs(activeStep - index) < totalSteps ? <>{icon}</> : null}
             </swiper-slide>
           ))}
         </swiper-container>
       </SwiperBox>
-      <Typography variant="h4" color={COLORS.white} sx={{ marginBottom: 5 }}>
+      <Typography
+        variant="h4"
+        color={COLORS.white}
+        sx={theme => ({ marginBottom: 5, color: theme.palette.layout.main })}
```

Comment:
Probably meant to remove `color={COLORS.white}`

Reply:
Refactored in https://github.com/oasisprotocol/explorer/pull/605/commits/cb393fe65d117dd9a145f35b5f179ed2d916f12c so that parent specifying light background also specifies dark text